mod-record-specifications-server/src/main/java/org/folio/rspec/domain/dto/ErrorCode.java

```diff
@@ -10,6 +10,8 @@ public enum ErrorCode {
   INVALID_REQUEST_PARAMETER("invalid-request-parameter", "103", null),
   DUPLICATE_FIELD_TAG("duplicate-specification-field-tag", "104", "specification.field.tag.duplicate"),
   SPECIFICATION_FETCH_FAILED("specification-fetch-failed", "105", "specification.fetch.failed"),
+  DUPLICATE_FIELD_INDICATOR("duplicate-field-indicator", "106", "field.indicator.order.duplicate"),
+  DUPLICATE_INDICATOR_CODE("duplicate-indicator-code", "107", "indicator.code.code.duplicate"),
```

Comment:
Is this 'code.code' intentional?

Reply:
Yes. Looks strange, but 1'st code is for entity, 2'd for entity field